### PR TITLE
Added SEO keywords

### PR DESCRIPTION
**Pull request by kirstenweis**
Added keywords to the title, subtitle & metadescription

### CODE REVIEW

File: _posts/plotly2/2016-06-23-how-to-save-export-and-share.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 title: How to Save, Export, and Share in Plotly 2.0
-subtitle: How to save your work and share it.
+subtitle: Learn to save and share your data visualizations
 permalink: /save-share-and-export-in-plotly/
 imageurl: ../static/images/how-to-save-export-and-share/save-thumb.png
 state: active
 tags: fundamentals
 order: 4
-meta_description: A tutorial on how to save, export, and share in Plotly 2.0.
+meta_description: A tutorial on how to save, export and share graphs in Plotly 2.0.
```
Comment (cldougl) [inline _posts/plotly2/2016-06-23-how-to-save-export-and-share.md]: Hey @kirstenweis what's the best practice for periods at the end of the the `meta_description` and `subtitle` fields? 
Across our tutorials it looks like some have it and some don't. Looks like you're mostly removing them but @chriddyp made a [pr](https://github.com/plotly/plotly.github.io/pull/201/commits/bf1c39d7f0dd762c560ddef822fbbc255ea6265a) not long ago to add a . to the `meta_description`

Comment (kirstenweis) [inline _posts/plotly2/2016-06-23-how-to-save-export-and-share.md]: Hey there... sorry for the late reply. I would treat meta descriptions as full sentences and add a period at the end. Subtitles generally don't have periods at the end.

Comment (cldougl) [inline _posts/plotly2/2016-06-23-how-to-save-export-and-share.md]: cool sounds good!

**Comment by cldougl** [comment]
This looks good to merge @kirstenweis  - let me know if you want to, or want me to do that

**Pull request merged**